cloudpickle/__init__.py

```diff
@@ -1,9 +1,10 @@
 from __future__ import absolute_import
 
 import sys
+import pickle
 
 
-if sys.version_info[:2] >= (3, 8):
+if hasattr(pickle.Pickler, 'global_hook'):
```

Comment:
Please add a comment mentioning that this is a new feature introduced in Python 3.8.